deploy/metrics-server-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metrics-server
+  namespace: kube-system
```

Comment:
> @piosz I think the point is -- "heapster runs in kube-system which is suboptimal. Let's change that when we do this disruptive change anyway"

Correct.  Separating components into different namespaces helps us reason about which areas have access to particular escalation powers.  `kube-system` is a super-powered namespace that heapster doesn't need access to.  The final goal is zero pods in `kube-system`.